nf_core/lint/actions_schema_validation.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python
+
+import logging
+import yaml
+import json
+import jsonschema
+import os
+import glob
+import requests
+
+
+def actions_schema_validation(self):
+    """
+    Validate workflows against a schema
+
+    Uses the JSON Schema from
+    https://json.schemastore.org/github-workflow
```

Comment:
Needs to be valid RST to make the hyperlink work in the docs.

Would also be good to add a bit more detail here - which files exactly? And where to get help (the URL is just a big JSON document). Have to assume that people reading this might not know that these files even exist..

Reply:
Aaah sorry, didn't think about that this will be rendered at that moment 😱 
Will write a nicer message, thanks!